FueledUtils/KeyboardInsetHelper.swift

```diff
@@ -1,10 +1,18 @@
 import Foundation
 import UIKit
 
+/// Binds keyboard appearance and metrics to scroll view content and scroll bar insets and/or a layout constraint \
+/// relative to reference view. This object is can to be created and linked in a sotryboard.
```

Comment:
Little typos here too I think. `This object can be created and linked in a storyboard.` ?